tests/integration/common/s2n_test_common.py

```diff
@@ -46,6 +53,13 @@ def get_process(cmd):
     return subprocess.Popen(cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
 
 
+def assert_no_errors(process):
```

Comment:
When I test out the working certificate, even with the `-CAfile <path to server pem>` argument, I get stderr output:

```
depth=0 C = AU, ST = Some-State, O = Internet Widgits Pty Ltd
verify return:1
```
There may be an easy way to not print this, but I haven't seen it in OpenSSL's cli.
Perhaps we could add a suppression list argument to this function?